modules/servodyn/src/ServoDyn_Registry.txt

```diff
@@ -439,6 +439,7 @@ typedef	^	ParameterType	IntKi	StCCmode	-	-	-	"Structural control control mode {0
 typedef	^	ParameterType	IntKi	NumOuts	-	-	-	"Number of parameters in the output list (number of outputs requested)"	-
 typedef	^	ParameterType	IntKi	NumOuts_DLL	-	-	-	"Number of logging channels output from the DLL (set at initialization)"	-
 typedef	^	ParameterType	CHARACTER(1024)	RootName	-	-	-	"RootName for writing output files"	-
+typedef	^	ParameterType	CHARACTER(1024)	PriPath  	-	-	-	"Path of the primary SD input file "	-
```

Comment:
`SD` is `SubDyn`, so this should be changed to `SrvD`.

Reply:
Corrected directly on rc-3.5.3 with commit 7a2c43e8b